src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -8,6 +8,7 @@
 using System.IO.Enumeration;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Collections.ObjectModel;
```

Comment:
nit: i read on [coding style](https://github.com/dotnet/corefx/blob/aca57537adf4015a8a69e0ae386970d45bd60c9a/Documentation/coding-guidelines/coding-style.md) should be sorted alphabetically right?